src/cloud_common/protocols/unit/certificates.py

```diff
@@ -0,0 +1,223 @@
+#
+#  Copyright (c) 2018-2024 Renesas Inc.
+#  Copyright (c) 2018-2024 EPAM Systems Inc.
+#
+import base64
+from typing import Annotated, Literal
+
+from pydantic import BaseModel, Field, field_serializer
+
+from cloud_common.protocols.unit.types import (
+    AosSensitiveBytes,
+    TypeCertificatesType,
+    TypeNodeIdMandatory,
+    TypeStatusForNonExecutables,
+)
+
+
+class AosCertificateIdentification(BaseModel):
+    """Certificate data."""
+
+    type: TypeCertificatesType
+    node_id: TypeNodeIdMandatory
+
+
+class AosUnitSecretData(BaseModel):
+    """Keeps unit secret used to decode secure device password."""
+
+    owner_password: Annotated[
+        AosSensitiveBytes,
+        Field(
+            alias='ownerPassword',
+            title='Owner Password',
+            description='The owner password.',
+        ),
+    ]
+
+    @field_serializer('owner_password', when_used='json')
+    def dump_secret(self, struct_value):
+        return base64.b64encode(struct_value.get_secret_value())
+
+
+class AosUnitSecret(BaseModel):
+    """Keeps unit secret used to decode secure device password."""
+
+    version: Annotated[
+        int,
+        Field(
+            alias='version',
+            title='Version',
+            description='Version of the unit secrets.',
+        ),
+    ]
+
+    data: Annotated[  # noqa: WPS110
+        AosUnitSecretData,
+        Field(
+            alias='data',
+            title='Data of the unit secret.',
+            description='Data of the unit secret.',
+        ),
+    ]
+
+
+class AosRenewCertsNotification(BaseModel):
+    """
+    AosUnit protocol: 'renewCertificatesNotification' message.
```

Comment:
Use same tamplate for comments:
"""IssueCertData issue certificate data."""
"""Keeps unit secret used to decode secure device password."""
"""AosUnit protocol: 'issuedUnitCertificates' message."""